includes/VerifiedWikiImporter.php

```diff
@@ -1260,15 +1123,7 @@ private function processRevision( $pageInfo, $revisionInfo ) {
 		}
 		$revision->setNoUpdates( $this->mNoUpdates );
 
-		// Callback
-		$out = $this->revisionCallback( $revision );
-
-		// Aqua modification
-		// We need to do this after the callback, which is `importRevision`,
-		// because we need the newly generated revision id.
-		$this->processVerification( $revisionInfo, $title );
```

Comment:
@JeroenDeDauw while testing this PR, it failed, because in the old verification, `processVerification()` is intentionally done after `$this->revisionCallback`. That `revisionCallback` does the hash writer hook stuff to insert the initial version of the verification data, which is then amended by `processVerification`. I suppose there is just no way around it. So I will keep the `VerifiedWikiImporter` for now.

Reply:
Actually, I am waiting for your feedback, if there is a better way to ensure the order of actions is preserved. I.e. `processVerification` needs to be the last step of processing one verification.

Reply:
So the two used are called in the wrong order for your usecase?

Reply:
Yes, for this PR, the order is wrong. Let me push my new changes to this PR.

Reply:
I'd be quite surprised if there is no way to get things to work. That said, this will require some focused dev work.